config/boot.rb

```diff
@@ -31,10 +31,16 @@
 #   include Padrino::Helpers::TranslationHelpers
 # end
 
+Padrino::Logger::Config[:production][:stream] = :stdout
+Padrino::Logger::Config[:production][:log_level] = :info
+Padrino::Logger::Config[:test][:stream] = :stdout
+Padrino::Logger::Config[:test][:log_level] = :info
+
 ##
 # Add your before (RE)load hooks here
 #
 Padrino.before_load do
+  Time.zone = 'UTC'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.